translation of gatsby-on-windows (#141)
gatsbyjs/gatsby-pt-BR: docs/docs/gatsby-on-windows.md

npm install
```

> Para o Visual Studio 2017, baixe [Visual Studio Community 2017](https://visualstudio.microsoft.com/vs/community/) e instale o desenvolvimento Desktop com fluxo de trabalho C++. Você pode desmarcar todo o resto.

Reviewer: Só queria comentar que eu não sei se faz sentido traduzir `Desktop development with C++` já que é uma opção pra ser selecionada. Mas como eu não sei a tradução dela, estou indeciso. O que tu acha?

Author: ah sim, já que é uma opção, é melhor deixar em itálico pra sinalizar que é um texto retirado de outro lugar.

npm install
```

Você deve estar com tudo pronto.

Se isso ainda não funcionar, consulte a
[página inicial do pacote`node-gyp` npm](https://www.npmjs.com/package/node-gyp) para

> instruções adicionais, entre em contato com a equipe `node-gyp` em

Reviewer: ```suggestion
instruções adicionais e entre em contato com a equipe `node-gyp` em
```

[GitHub](https://github.com/nodejs/node-gyp/issues).

## gatsby-plugin-sharp precisa do Node x64

> Alguns plugins que dependem de dependências nativas do NPM requerem a compilação Node x64 do Node.js. Se você estiver com dificuldades para instalar o gatsby-plugin-sharp, tente instalar o Node x64 e remover o `node_modules` e executar o` npm install`.

Reviewer: ```suggestion
Alguns plugins que dependem de dependências nativas do NPM requerem o _build_ Node x64 do Node.js. Se você estiver com dificuldades para instalar o gatsby-plugin-sharp, tente instalar o Node x64, remover o `node_modules`e executar o` npm install`.
```
Acho melhor não traduzir, o que tu acha?

Author: ta bem, faz sentido o _build_ aqui.

## gatsby-plugin-sharp requer libvips

> Sharp usa uma biblioteca C, libvips. Se você estiver tendo problemas ao instalar o Sharp, tente remover `C:\Usuários\[usuário]\AppData\Roaming\npm-cache\_libvips`.

Reviewer: ```suggestion
Sharp usa uma biblioteca C, libvips. Se você estiver tendo problemas ao instalar o Sharp, tente remover `C:\Users\[usuário]\AppData\Roaming\npm-cache\_libvips`.
```

## Subsistema do Windows para Linux

> Se a instalação de dependências ou o desenvolvimento no Windows em geral causar dores de cabeça, o Windows 10 oferece uma ótima alternativa: [Subsistema do Windows para Linux](https://docs.microsoft.com/en-us/windows/wsl/about). Permite executar a maioria das ferramentas, utilitários e aplicativos de linha de comando em um ambiente GNU/Linux diretamente no Windows, sem modificação, sem a sobrecarga de uma máquina virtual. No cenário acima, você faria o download, por exemplo. Ubuntu, abra o terminal, [instale o Node](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) e execute `sudo apt-get install build -essencial` no terminal - e a compilação funciona de forma muito mais confiável. Observe que você deve excluir qualquer pasta `node_modules` existente no seu projeto e reinstalar as dependências no seu ambiente WSL.

Reviewer: ```suggestion
Se a instalação de dependências ou o desenvolvimento no Windows em geral causar dores de cabeça, o Windows 10 oferece uma ótima alternativa: [Subsistema do Windows para Linux](https://docs.microsoft.com/en-us/windows/wsl/about). Permite executar a maioria das ferramentas, utilitários e aplicativos de linha de comando em um ambiente GNU/Linux diretamente no Windows, sem modificação ou sobrecarga de uma máquina virtual. No cenário acima, você poderia fazer o download, por exemplo, do Ubuntu. Para isso, você deve abrir o terminal, [instalar o Node](https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions) e executar `sudo apt-get install build -essencial` no terminal - gerando uma compilação muito mais segura. Observe que você deve excluir qualquer pasta `node_modules` existente no seu projeto e reinstalar as dependências no seu ambiente WSL.
```
Tentei corrigir a tradução, porque lendo em português ficou um pouco desconectado. O que tu acha?

Author: gostei da sua sugestão
